doc/pages/architecture/080-mqtt-v1-protocol.md

```diff
@@ -0,0 +1,139 @@
+# Astarte MQTT v1 Protocol
+
+Astarte MQTT v1 Protocol allows communication between devices and Astarte. It is the first protocol that has been implemented in Astarte, and it has been designed with Astarte requirements in mind.
```

Comment:
We have to specify that this protocol makes several assumptions about the capabilities of the transport, and that its reference implementation is in the Astarte/VerneMQ Transport.

Reply:
We don't need to tell anything about how it works the broker, we just tell users that they must use MQTT v3.1.1

Reply:
And that's exactly why we need to add this. For example, without mutual SSL auth the whole thing won't work